src/content/tutorials/user-auth-with-auth0/_index.md

```diff
@@ -0,0 +1,797 @@
+---
+title: 'User Auth with Auth0'
+showNew: true
+---
+
+In this tutorial, you'll integrate Auth0, an identity management platform, into a Cloudflare Workers application. Adding authorization and authentication to an application is a common task for developers. By implementing it using Cloudflare Workers, you can take advantage of Workers' unique platform advantages to simplify how and when your application needs user data.
+
+## What you'll learn
+
+- How to authorize and authenticate users in Workers
+- How to persist authorization credentials inside of Workers KV
+- How to use Auth0 user info inside of your Workers application
+
+## Prerequisites
+
+To publish your Worker to Cloudflare, you'll need a few things:
+
+- A Cloudflare account, and access to the API keys for that account
```

Comment:
I believe this needs to be a "paid" account. We should probably call this out or discuss how to work around this.